src/modules/AuthenticationModule/components/Verify Account/VerifyAccount.tsx

```diff
@@ -1,8 +1,103 @@
+                                <form onSubmit={handleSubmit(onSubmit)} className=" pt-4 pb-5">
+                                    <div className="row justify-content-evenly">
+                                        <div className="col-md-11">
+                                            <div className="input-group pb-3">
+                                                <p className="my-1 register-text-color">E-mail</p>
+                                                <input
+                                                    type="text"
+                                                    className="input-style"
+                                                    placeholder="Enter your E-mail"
+                                                    {...register("email", {
+                                                        required: "Email is required",
+                                                        pattern: {
+                                                            value: /^[\w-.]+@([\w-]+\.)+[\w-]{2,4}$/g,
+                                                            message: "Invalid mail",
+                                                        },
+                                                    })}
+                                                />
+                                            </div>
+                                            {errors.email && (
+                                                <p className="alert alert-danger">
+                                                    {errors.email.message}
+                                                </p>
+                                            )}
+
+                                            <div className="input-group pb-3">
+                                                <p className="my-1 register-text-color">OTP Verfication</p>
+                                                <input
+                                                    type="text"
+                                                    className="input-style"
+                                                    placeholder="Enter OTP Code"
+                                                    {...register("code", {
+                                                        required: "OTP is required",
+                                                    })}
+                                                />
+                                            </div>
+                                            {errors.phoneNumber && (
+                                                <p className="alert alert-danger">
+                                                    {errors.code.message}
+                                                </p>
+                                            )}
+                                        </div>
+                                    </div>
+                                    <div className="d-flex justify-content-center">
```

Comment:
We need to disable the button while submitting to prevent sending unnecessary requests.